app/views/dashboard/autoyast.xml.erb

```diff
@@ -10,6 +10,7 @@
         <source><![CDATA[
         #!/bin/sh
         echo "master: <%= @controller_node %>" > /etc/salt/minion.d/master.conf
+        echo "grains_refresh_every: 10" >> /etc/salt/minion.d/master.conf
```

Comment:
Why not adding it to another file like `/etc/salt/minion.d/grains_refresh.conf`?

I'm okay with adding this temporarily here, but shouldn't this configuration be managed by salt itself? We can trigger a restart of the salt-minion when this configuration changes, in theory we only need to set the master here.